### PR TITLE
DCA-335: [BB JMeter] Hide `chmod 400 ssh key` action from results output

### PR DESCRIPTION
**Pull request by mmizin**
https://bulldog.internal.atlassian.com/browse/DCA-335

### CODE REVIEW

File: app/jmeter/bitbucket.jmx [hunk 1]
```diff
@@ -261,7 +261,7 @@ log.info(&quot;GIT_LOG_DIR: &quot; + gitLogDir);
           <SystemSampler guiclass="SystemSamplerGui" testclass="SystemSampler" testname="Chmod 400 ssh key" enabled="true">
             <boolProp name="SystemSampler.checkReturnCode">false</boolProp>
             <stringProp name="SystemSampler.expectedReturnCode">0</stringProp>
-            <stringProp name="SystemSampler.command">chmod</stringProp>
+            <stringProp name="SystemSampler.command">2chmod</stringProp>
```
Comment (astashys) [inline app/jmeter/bitbucket.jmx]: really "2chmod" ? =)

File: app/jmeter/bitbucket.jmx [hunk 2]
```diff
@@ -281,7 +281,23 @@ log.info(&quot;GIT_LOG_DIR: &quot; + gitLogDir);
...
+              <stringProp name="scriptLanguage">groovy</stringProp>
+              <stringProp name="parameters"></stringProp>
+              <stringProp name="filename"></stringProp>
+              <stringProp name="cacheKey">true</stringProp>
+              <stringProp name="script">if(!prev.isSuccessful()){
+	log.error(&quot;Error occured in Thread name::&quot; + prev.getThreadName() + &quot;::Sampler name::&quot; + prev.getSampleLabel() + &quot;::&quot; + prev.getResponseMessage() + &quot;::&quot; + prev.getResponseDataAsString())
+	ctx.getEngine().askThreadsToStop();
+}	
+else{
+	prev.setIgnore()
```
Comment (astashys) [inline app/jmeter/bitbucket.jmx]: let's add here comment why we are doing this

File: app/jmeter/bitbucket.jmx (hunk 1 above)
Comment (mmizin) [reply app/jmeter/bitbucket.jmx]: Thx, already changed!

File: app/jmeter/bitbucket.jmx [hunk 3]
```diff
@@ -281,7 +281,23 @@ log.info(&quot;GIT_LOG_DIR: &quot; + gitLogDir);
...
           </SystemSampler>
-          <hashTree/>
+          <hashTree>
+            <JSR223PostProcessor guiclass="TestBeanGUI" testclass="JSR223PostProcessor" testname="Ignore OS Process Sampler result" enabled="true">
+              <stringProp name="scriptLanguage">groovy</stringProp>
+              <stringProp name="parameters"></stringProp>
+              <stringProp name="filename"></stringProp>
+              <stringProp name="cacheKey">true</stringProp>
+              <stringProp name="script">if(!prev.isSuccessful()){
+	log.error(&quot;Error occured in Thread name::&quot; + prev.getThreadName() + &quot;::Sampler name::&quot; + prev.getSampleLabel() + &quot;::&quot; + prev.getResponseMessage() + &quot;::&quot; + prev.getResponseDataAsString())
```
Comment (astashys) [inline app/jmeter/bitbucket.jmx]: `// We want to hide the sampler if it passes successfully to not output in results and show if it fails for troubleshooting purpose`

**Pull request merged**